.github/workflows/aem-sync-to-cloudmanager-repo.yml

```diff
@@ -0,0 +1,228 @@
+name: 📦 AEM Sync to Cloud Manager Repo
+
+on:
+  workflow_call:
+    inputs:
+      # Build Configuration
+      skip-tests:
+        description: "Skip Maven test execution"
+        type: boolean
+        required: false
+        default: false
+      skip-build:
+        description: "Skip Maven build execution"
+        type: boolean
+        required: false
+        default: false
+
+      # Deployment Configuration
+      deploy-to-cloudmanager:
+        description: "Deploy to AEM Cloud Manager repository"
+        type: boolean
+        required: false
+        default: true
+      remote-branch:
+        description: "Target branch name in Cloud Manager (optional, defaults to source branch name)"
+        type: string
+        required: false
+        default: ""
+
+      # Advanced Configuration
+      maven-version:
+        description: "Maven Docker image version (e.g., 3.8.6-openjdk-11)"
+        type: string
+        required: false
+        default: "3.8.6-openjdk-11"
+      debug:
+        description: "Enable verbose logging and debug output"
+        type: boolean
+        required: false
+        default: false
+
+    secrets:
+      cm-git-user:
+        description: "Cloud Manager Git username"
+        required: true
+      cm-git-password:
+        description: "Cloud Manager Git password or access token"
+        required: true
+      cm-git-remote:
+        description: "Cloud Manager Git remote URL (without https://)"
+        required: true
+
+    outputs:
+      sync-status:
+        description: "Status of the Cloud Manager sync (success/skipped/failed)"
+        value: ${{ jobs.sync-to-cloudmanager.outputs.status }}
+
+jobs:
+  run-tests:
+    name: 🧪 Run Tests
+    runs-on: ubuntu-latest
+    if: inputs.skip-tests == false
+
+    container:
+      image: maven:${{ inputs.maven-version }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Cache Maven packages
+        if: ${{ !env.ACT }}  # Skip when running with act locally
+        uses: actions/cache@v4
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+
+      - name: Run Maven tests
+        run: |
+          echo "🧪 Running Maven tests..."
+
+          verbose=""
+          if [ "${{ inputs.debug }}" = "true" ]; then
+            verbose="-X"
+          fi
+
+          mvn -B verify $verbose
+          echo "✅ Tests completed successfully"
+
+  build:
+    name: 🔨 Build
+    runs-on: ubuntu-latest
+    needs: [run-tests]
+    if: |
+      always() &&
```

Comment:
Maybe I'm misunderstanding `always()` but what's the point of this? 😅 Is this not the equivalent of running `if true && condition`?